default.nix

```diff
@@ -0,0 +1,48 @@
+{ pkgs ? import <nixpkgs> {} }:
+
+with pkgs;
+
+let
+  version = "0.9.3";
+
+  nodegit = import ./nodegit.nix { inherit pkgs; };
+
+  libs = [ stdenv.cc.cc glibc ];
+
+in
+
+  stdenv.mkDerivation {
+    name = "clever-cli-${version}";
```

Comment:
I saw that you were importing the package as `clever-tools`.
Do we have a good reason to name it differently (`clever-cli`) here ?

Reply:
No I don't. Remember I don't know what I'm doing here.